src/components/graphCard/graphCardChartLegend.js

```diff
@@ -116,8 +90,6 @@ class GraphCardChartLegend extends React.Component {
           position={TooltipPosition.top}
           enableFlip={false}
           distance={5}
-          entryDelay={100}
```

Comment:
optional: may want to consider adding this delay back in, tooltip fires almost immediately in some cases making it difficult to read the graph itself